### DISCUSSION TITLE
Training on Multiple Target Words

### DISCUSSION THREAD

**Discussion by ADHDSquir** [dscripka/openWakeWord#165] [Q&A] [answered]
In the [Home Assistant WakeWords Collection](https://github.com/fwartner/home-assistant-wakewords-collection/) I see multiple wake words that say they react to multiple variations and/or were trained on multiple target words. For example, [Hey Kitt](https://github.com/fwartner/home-assistant-wakewords-collection/tree/main/en/hey_kitt) says the tflite model reacts to `Hey Kitt` and `Okay Kitt` and that it was trained with `target_words = ["hey kitt", "hey kitt!", "okay kitt", "okay kitt!"]`. How is this achieved? Everything I have seen is limited to training only a single version of a single phrase.

**Comment by dscripka** [comment, marked as answer]
See [this related issue](https://github.com/dscripka/openWakeWord/issues/129) for how to train models that activate to multiple phrases.

Note that this is more advanced functionality that the basic automatic model training notebook does not directly support. However, by creating your own training config file as shown in the issue, you can train a model with this capability.

**Reply by ADHDSquir** [reply]
Perfect, thank you! I had looked at the more complete notebook and the config but must not have read close enough.
